samples/apps/copilot-chat-app/webapp/src/components/chat/chat-history/ChatHistoryItem.tsx

```diff
@@ -93,11 +98,17 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = ({ message, getRe
         content = <ChatHistoryTextContent message={message} />;
     }
 
+    const contentAsString = (message.content as string).trim()
+                            .replace(/^sk:\/\/.*$/gm, (match: string) => createCommandLink(match))
+                            .replace(/^!sk:.*$/gm, (match: string) => createCommandLink(match));
```

Comment:
since `ChatHistoryTextContent` is already performing this logic as well, can we create a shared helper fn that is used in both?
imo it makes sense to put it and `createCommandLink` both in `utils/TextUtils.ts`.

Reply:
or better yet, could the `data-content` attribute just exist on the `div` in `ChatHistoryTextContent`?

Reply:
Moved the functions into the text util.
I will leave the data-content on the div its currently on, since we already find and query it for things and it is the container for the ChatHistoryTextContent.